etc/qbittorrent.profile

```diff
@@ -34,11 +34,12 @@ nogroups
 nonewprivs
 noroot
 nosound
-protocol unix,inet,inet6
+protocol unix,inet,inet6,netlink
```

Comment:
Does qbittorrent work without netlink (despite the error message) or does it break using qbittorrent?